src/main/java/org/jitsi/videobridge/DtlsTransport.java

```diff
@@ -479,6 +495,17 @@ protected void onIceFailed()
         endpoint.getConference().getStatistics().hasIceFailedEndpoint = true;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    protected void onIceConsentUpdated(long time)
+    {
+       super.onIceConsentUpdated(time);
+       packetIOActivity.setLastPacketReceivedTimestampMs(
+           System.currentTimeMillis());
+    }
+
```

Comment:
The `time` parameter here is the `currentTimeMillis()` when the consent packet was received, so the additional call should be unnecessary.

Reply:
@JonathanLennox ok, I'll update. But the generic idea here was to not overwrite timestamp which is already newer and I assumed that System.currentTimeMillis will not go back in history, which is actually incorrect assumption, but used everywhere in a codebase... Just if-check before assignment is not enough due to field might be assigned from multiple threads. Ideally monotonic `System.nanoTime` should be used here and many other places in the project , which should guarantee non-decreasing and monotonic timestamps. Anyway changing now to just assigning `time`.